### PR TITLE
Upgrade to quarkus-cxf 2.0.0.Alpha5, Add back XML Security Sign

### PR DESCRIPTION
**Pull request by ppalaga**
Fix https://github.com/apache/camel-quarkus/issues/4684

### CODE REVIEW

**Review by zbendhiba — APPROVED** [review]

**Comment by zbendhiba** [comment]
@ppalaga  Can you please update this migration guide https://camel.apache.org/camel-quarkus/3.0.x/migration-guide/3.0.0.html

**Review by oscerd — APPROVED** [review]

**Comment by ppalaga** [comment]
> @ppalaga Can you please update this migration guide https://camel.apache.org/camel-quarkus/3.0.x/migration-guide/3.0.0.html

Thanks for the reminder, let me do it!

**Comment by ppalaga** [comment]
> > @ppalaga Can you please update this migration guide https://camel.apache.org/camel-quarkus/3.0.x/migration-guide/3.0.0.html
> 
Done

**Comment by zbendhiba** [comment]
@ppalaga thanks
Sorry another thing is missing, the new icon-url is missing in this file : https://github.com/apache/camel-quarkus/blob/main/extensions/xmlsecurity/runtime/src/main/resources/META-INF/quarkus-extension.yaml

BTW, will this be generated automatically when adding a new extension?

**Comment by ppalaga** [comment]
> Sorry another thing is missing, the new icon-url is missing in this file : https://github.com/apache/camel-quarkus/blob/main/extensions/xmlsecurity/runtime/src/main/resources/META-INF/quarkus-extension.yaml

Fixed along with a few other small things.

> will this be generated automatically when adding a new extension?

Yes it is driven by the template in [tooling/create-extension-templates/quarkus-extension.yaml#L26](https://github.com/apache/camel-quarkus/blob/main/tooling/create-extension-templates/quarkus-extension.yaml#L26)

File: extensions/datasonnet/runtime/src/main/resources/META-INF/quarkus-extension.yaml [hunk 1]
```diff
@@ -23,6 +23,7 @@
 ---
 name: "Camel DataSonnet"
 description: "To use DataSonnet scripts for message transformations"
+icon-url: "https://camel.apache.org/_/img/logo-d-f21b25ba38.svg"
```
Comment (zbendhiba) [inline extensions/datasonnet/runtime/src/main/resources/META-INF/quarkus-extension.yaml:26]: Nice catch

**Comment by zbendhiba** [comment]
> > will this be generated automatically when adding a new extension?
> 
> Yes it is driven by the template in [tooling/create-extension-templates/quarkus-extension.yaml#L26](https://github.com/apache/camel-quarkus/blob/main/tooling/create-extension-templates/quarkus-extension.yaml#L26)

Awesome! Thanks

**Pull request merged**